cli/internal/cmd/spinner.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	spinnerStates = []string{"⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷"}
+	dotsStates    = []string{".", "..", "..."}
+)
+
+type Spinner struct {
```

Comment:
```suggestion
type spinner struct {
```
As this is only used within the `cmd` package, it can be private.